sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareResult.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// DocumentHealthcareEntities.
+    /// </summary>
+    public partial class DocumentHealthcareResult
+    {
+        internal DocumentHealthcareResult(DocumentHealthcareEntitiesInternal documentHealthcareEntities)
+        {
+            Entities = documentHealthcareEntities.Entities;
+            Relations = documentHealthcareEntities.Relations != null ? ResolveHealthcareRelations(documentHealthcareEntities.Entities, documentHealthcareEntities.Relations) : null;
```

Comment:
according to the swagger, relations is required so it will never be empty. Are u seeing relations not returned from the service?

Reply:
yes i saw no relations in few requests

Reply:
interesting.. then is this a service bug? or does the swagger need to be updated?

Reply:
The service is not returning relations or warnings if there are not present.